Pathogens/Utility/location_utils/mapbox_request_param_overrides.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
Although I strongly appreciate the work it took to do this, I feel like this may be an issue that needs to be solved on a data level. I am OK to keep this in for now (if you agree) but would prefer to instead maybe update the ETL so that it generated a report that found faulty data (not sure if this was all done manually, and if so kudos to you! I think it would be an interesting problem to solve on checking if the city state and country values are valid or not. We can then pass this report onto the data team and have them make changes to the data as needed. Once the changes are made we can run the etl again. 

Let me know what your thoughts on this are.

In a situation where these happened for specific edge cases (and few) I would be OK hardcoding a solution but this is not maintainable in the long run and should not be on us as dev's to be fixing dirty data in a way that is extra complex. 

Ideally we never want to have an "overrides" file. I would not even be opposed to looking for an alternative API but since this works for 90% of our data and fails on some, it probably is an issue with the data and not the API.

Reply:
Everything you've written makes sense to me.

I gave some thought to the idea of a report that lists all the problematic data in the ETL and based on my understanding of the mapbox API that sounds viable to me so I'll give it a go and see where we get :)

Reply:
Alright my man, one report that generates possible faulty lat lng data for us to analyze and bring to the data team for them to change in Airtable coming right up :man_cook: https://github.com/serotracker/iit-backend/pull/342

I know that PR is a bit confusing, I tried to annotate it to the best of my ability but some of it kind of feels like rambles with not enough explanation which I'm sorry for but it's hard to summarize all of the warning signs that the data is bad, all of the different ways mapbox gives it's data, and how I believe the mapbox API is working. If you don't understand any part of that, let's hop on a call, I'm super down to talk about this there's just a lot going on and I need to know what part people are getting stuck on :)

Reply:
Great idea with the report btw :+1: